components/ban-search.js

```diff
@@ -59,7 +59,7 @@ function BanSearch() {
   }, [results])
 
   useEffect(() => {
-    if (input) {
+    if (input && input.length > 2) {
```

Comment:
`input` est toujours défini, il n'est donc pas nécessaire de vérifier. La condition peut-être plus claire avec `>= 3`.

```suggestion
    if (input.length >= 3) {
```

Attention, le message d'erreur de l'API indique : `"q must contain at least 3 chars and start with a number or a letter"`. Les premiers caractères ne sont pas vérifiés et il est donc possible de faire une requête avec uniquement des espaces par exemple.